content/ar/docs/setup/best-practices/certificates.md

```diff
@@ -0,0 +1,281 @@
+| ca.crt,key                     | kubernetes-ca                     | Kubernetes general CA                                                                 |
+| etcd/ca.crt,key                | etcd-ca                           | For all etcd-related functions                                                        |
+| front-proxy-ca.crt,key         | kubernetes-front-proxy-ca         | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
+
+
+بالإضافة إلى شهادات الاعتماد (CA) المذكورة أعلاه، من الضروري أيضًا الحصول على زوج مفتاح عام/خاص لإدارة حساب الخدمة، `sa.key` و `sa.pub`.
+المثال التالي يوضح ملفات المفاتيح والشهادات CA المبينة في الجدول السابق:
+
+
+```
+/etc/kubernetes/pki/ca.crt
+/etc/kubernetes/pki/ca.key
+/etc/kubernetes/pki/etcd/ca.crt
+/etc/kubernetes/pki/etcd/ca.key
+/etc/kubernetes/pki/front-proxy-ca.crt
+/etc/kubernetes/pki/front-proxy-ca.key
+```
+
+
+
+
+### كل الشهادات
+
+إذا لم ترغب في نسخ مفاتيح CA الخاصة إلى العنقود الخاصة بك، يمكنك إنشاء جميع الشهادات بنفسك.
+
+
+الشهادات المطلوبة:
+
+
+
+|          الاسم المميز الافتراضي             |    الشهادة الأم  |           إسم المالك (في الموضوع) |                    النوع  |               المستضيفين (SAN)                                                 |
+|-----------------------------------|---------------------------|-----------------------------------|------------------|----------------------------------------------------------------------------------------|
+| kube-etcd                         | etcd-ca                   |                                   | خادم, مستخدم     |  <div dir="ltr">    `<hostname>` , `<Host_IP>`, `localhost`, `127.0.0.1`    </div>     |
+| kube-etcd-peer                    | etcd-ca                   |                                   | خادم, مستخدم     |  <div dir="ltr">    `<hostname>` , `<Host_IP>`, `localhost`, `127.0.0.1`    </div>     |
+| kube-etcd-healthcheck-client      | etcd-ca                   |                                   | مستخدم           |                                                                                        |
+| kube-apiserver-etcd-client        | etcd-ca                   |                                   | مستخدم           |                                                                                        |
+| kube-apiserver                    | kubernetes-ca             |                                   | خادم             |  <div dir="ltr">     `<hostname>`, `<Host_IP>`, `<advertise_IP>`, `[1]`    </div>      |
+| kube-apiserver-kubelet-client     | kubernetes-ca             | system:masters                    | مستخدم           |                                                                                        |
+| front-proxy-client                | kubernetes-front-proxy-ca |                                   | مستخدم           |                                                                                        |
```

Comment:
/hold

I recommend using a shortcode rather than embedding HTML like this. Using a shortcode is more maintainable.

If we want an entire table column to be LTR, we could also look at customising a shortcode at table level.

Also, the placeholder names (eg `<hostname>`) could be in Arabic, I think.
Something like: `<اسم المضيف>`

Reply:
OK to unhold this once the overall approach to `dir` overrides is settled.

Reply:
I did localize all the placeholder names, and now we don't need to use `dir="ltr"` on this file.

Reply:
adding the shortcode for that still a good feature we will file an issue an implement it when needed

Reply:
Define a ShortCode to overrides the LTR / RTL direction.
Issue : #46318

Reply:
Thank you ! Well defined